docs/reference/commands/disk.rst

```diff
@@ -33,13 +33,27 @@ Global flags:
 ``add``
 -------
 
-Adds a disk to the cluster, alongside optional devices for write-ahead logging and database management.
+Adds a new Ceph disk (OSD) to the cluster, alongside optional devices for write-ahead logging and database management.
+
+The command takes a parameter SPEC which is either a path to a block device such as /dev/sdb or a specification for one or more loop files.
+
+The specification for loop files is of the form "loop,<size>,<nr>"
+
+- size is a number with M, G, or T suffixes for megabytes, gigabytes, or terabytes.
+- nr is the number of file-backed loop OSDs to create.
+
+For instance, a spec of `loop,8G,3` will create 3 file-backed loop OSDs of 8GB each.
+
+While loop files provide a convenient way to quickly set up small test and development clusters, for
+production deployments it is **strongly** recommended to use dedicated block devices for safety and
+performance reasons.
```

Comment:
I feel that this paragraph should be put into the single-node tutorial.

Reply:
what is the safety concern with loopback devices? it's not clear to me, but if there is a security concern then we might want to spell that out

Reply:
i see you have also added this to the single-node file. same concerns but i would simply remove this content from this current **reference** file

Reply:
The safety concern is about OSD failure domains. If you use several file-backed OSDs they will necessarily live on a single physical disk. So even though you have e.g. 3xOSD your data will be gone if that single disk fails

Reply:
I'll remove this para here